packages/broker/test/integration/plugins/operator/MaintainOperatorValueService.test.ts

```diff
@@ -0,0 +1,133 @@
+            auth: {
+                privateKey: ADMIN_WALLET_PK
+            }
+        })
+        streamId1 = (await client.createStream(`/operatorvalueservicetest-1-${Date.now()}`)).id
+        streamId2 = (await client.createStream(`/operatorvalueservicetest-2-${Date.now()}`)).id
+        await client.destroy();
+
+        ({ operatorWallet, operatorContract } = await deployNewOperator())
+
+        await (
+            await token.connect(operatorWallet).transferAndCall(operatorContract.address, parseEther(`${STAKE_AMOUNT * 2}`), operatorWallet.address)
+        ).wait()
+        for (const streamId of [streamId1, streamId2]) {
+            const sponsorship = await deploySponsorship(config, operatorWallet, { streamId })
+            await (await token.connect(operatorWallet).transferAndCall(sponsorship.address, parseEther(`${SPONSOR_AMOUNT}`), "0x")).wait()
+            await (await operatorContract.stake(sponsorship.address, parseEther(`${STAKE_AMOUNT}`))).wait()
+            expect(await token.balanceOf(sponsorship.address)).toEqual(parseEther(`${SPONSOR_AMOUNT + STAKE_AMOUNT}`))
+        }
+    }, 60 * 1000)
+
+    it("updates both sponsorships to stay over the threshold", async () => {
+        const penaltyFraction = parseEther("0.001")
+        const threshold = penaltyFraction.mul(200).toBigInt()
+        const maintainOperatorValueService = new MaintainOperatorValueService(operatorConfig, penaltyFraction.toBigInt())
+
+        const totalValueInSponsorshipsBefore = await operatorContract.totalValueInSponsorshipsWei()
+
+        // wait for sponsorships to accumulate earnings so approximate values differ enough form the real values
+        await wait(3000)
+
+        maintainOperatorValueService.start()
+
+        await waitForCondition(async () => await operatorContract.totalValueInSponsorshipsWei() > totalValueInSponsorshipsBefore, 10000, 1000)
+        
+        const diff = await getDiffBetweenApproxAndRealValues()
+        expect((await operatorContract.totalValueInSponsorshipsWei()).toBigInt()).toBeGreaterThan(totalValueInSponsorshipsBefore.toBigInt())
+        expect(diff).toBeLessThan(threshold)
+
+        await maintainOperatorValueService.stop()
+    }, 60 * 1000)
+
+    it("needs only one sponsorship to stay over the threshold", async () => {
+        const penaltyFraction = parseEther("0.0005")
+        const threshold = penaltyFraction.mul(200).toBigInt()
+        const maintainOperatorValueService = new MaintainOperatorValueService(operatorConfig, penaltyFraction.toBigInt())
+
+        const totalValueInSponsorshipsBefore = await operatorContract.totalValueInSponsorshipsWei()
+
+        // wait for sponsorships to accumulate earnings so approximate values differ enough form the real values
+        await wait(3000)
+
+        maintainOperatorValueService.start()
+
+        await waitForCondition(async () => await operatorContract.totalValueInSponsorshipsWei() > totalValueInSponsorshipsBefore, 10000, 1000)
```

Comment:
What is the data type of `operatorContract.totalValueInSponsorshipsWei()`? We can do comparison operators to that with `>`, but it is not a `bigint` as we convert it `toBigInt` in 128?

Reply:
Will use ".gt" instead of "<"